controllers/gce/instances/instances.go

```diff
@@ -63,8 +63,7 @@ func (i *Instances) Init(zl zoneLister) {
 // all of which have the exact same named port.
 func (i *Instances) AddInstanceGroup(name string, port int64) ([]*compute.InstanceGroup, *compute.NamedPort, error) {
 	igs := []*compute.InstanceGroup{}
-	// TODO: move port naming to namer
-	namedPort := &compute.NamedPort{Name: fmt.Sprintf("port%v", port), Port: port}
+	namedPort := utils.GetNamedPort(port)
```

Comment:
In the next PR when passing all ports, can you please rename `AddInstanceGroup` to something like `SyncInstanceGroups`, please?

Reply:
Sure. How about `EnsureInstanceGroupsAndPorts`? Same as the corresponding cluster_manager method name.

Reply:
Sounds good!